signing_service/signing_service/utils.py

```diff
@@ -112,5 +115,41 @@ def __call__(  # type: ignore
 
 class Notifier(object):
 
-    def send(self) -> None:
-        pass
+    def __init__(
+        self,
+        from_email_address: str,
+        from_email_password: str,
+        to_email_addresses: list,
+    ) -> None:
+        self.from_email_address = from_email_address
+        self.from_email_password = from_email_password
+        self.to_email_addresses = to_email_addresses
+
+    def send(
+        self,
+        message: str,
+    ) -> None:
+        message_to_send = MIMEMultipart()
+        message_to_send['From'] = "Signing Service Notifier"
+        message_to_send['To'] = ','.join(self.to_email_addresses)
+        message_to_send['Subject'] = "Signing Service notification"
+        message_to_send.attach(MIMEText(message))
+        if self.from_email_address is not None and self.from_email_password is not None:
+            self.send_email(message_to_send)
+        print(message_to_send)
+        del message_to_send
```

Comment:
Is this necessary? I think garbage collector should handle this by default. If there is a reason please let me know.

Reply:
I've just seen it being use in some examples when `MIMEMultipart()` is used, I'm don't know what was the exact reason.

Reply:
I think it shouldn't be deleted explicitly. Leave it to python ;)

Reply:
Done